test/Interop/Cxx/stdlib/use-std-iterator.swift

```diff
@@ -0,0 +1,46 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
+//
+// REQUIRES: executable_test
+//
+// Enable this everywhere once we have a solution for modularizing libstdc++: rdar://87654514
+// REQUIRES: OS=macosx
+
+import StdlibUnittest
+import StdIterator
+import std.iterator
+import StdVector
+import std.vector
+
+var StdIteratorTestSuite = TestSuite("StdIterator")
+
+StdIteratorTestSuite.test("init") {
+    var vector = Vector()
+    var _1: CInt = 1
+    vector.push_back(&_1)
+    let it = vector.beginMutating()
+    expectEqual(it[0], 1)
+}
+
+StdIteratorTestSuite.test("next") {
+    var vector = Vector()
+    var _1: CInt = 1, _2: CInt = 2, _3: CInt = 3
+    vector.push_back(&_1)
+    vector.push_back(&_2)
+    vector.push_back(&_3)
+    let it = vector.beginMutating()
+    let n1 = next(it, 2)
```

Comment:
Can we call `std.__1.next`?

Reply:
I tried that and had the same result, `couldnt find next`